best-practices/check_deprecated_apis.yaml

```diff
@@ -13,41 +13,52 @@ metadata:
       Note that checking for some of these resources may require modifying the Kyverno
       ConfigMap to remove filters.
 spec:
-  validationFailureAction: audit
-  background: false
+  validationFailureAction: enforce
```

Comment:
Do we want to set default action to `enforce`?

Reply:
maybe for this policy, audit would be better? Just thinking for users who are on older k8s version, who aren't ready to use v1 but can still get the warning? Otherwise it would block them from using it at all

Reply:
I'd suggest setting it to `audit` by default as this won't impact anything. Users can configure it different if needed.